docs/java-rest/high-level/migration/get-deprecation-info.asciidoc

```diff
@@ -0,0 +1,34 @@
+--
+:api: get-deprecation-info
+:request: DeprecationInfoRequest
+:response: DeprecationInfoResponse
+--
+
+[id="{upid}-{api}"]
+=== Get Deprecation Info
+
+[id="{upid}-{api}-request"]
+==== Get Deprecation Info Request
+
+A +{request}+ can be applied to one or more indices:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests-file}[{api}-request]
+--------------------------------------------------
+<1> Create a new request instance
+
+include::../execution.asciidoc[]
+
+[id="{upid}-{api}-response"]
+==== Get Deprecation Info Response
+
+The returned +{response}+ contains the cluster, index and node level deprecations.
```

Comment:
This is awkward and not very clear, do you think we could reword it to something like:

```suggestion
The returned +{response}+ contains information about deprecated features currently in use at the cluster, node, and index level.
```

Reply:
woah what is this black magic commit suggestion thing? I kinda want to click it just to have played around with it!